src/rtf/src/TestSuit.cpp

```diff
@@ -26,12 +26,16 @@ TestSuit::~TestSuit() {
 }
 
 void TestSuit::addTest(RTF::Test* test) {
-    tests.insert(test);
+    tests.push_back(test);
```

Comment:
To maintain the same semantics, we should make sure that we don't add two times the same test pointer in the vector. Without this check, the `removeTest` would need to be improved.